tests/unit/test_backends.py

```diff
@@ -193,6 +156,22 @@ def setUp(self):
         self._dataDir = os.path.join("tests", "data")
         self._backend = backend.FileSystemBackend(self._dataDir)
 
+    def testDatasets(self):
+        self.assertEqual(self._backend.getNumDatasets(), 1)
+        dataset = self._backend.getDatasetByIndex(0)
+        self.assertEqual(dataset.getLocalId(), "dataset1")
+        self.assertEqual(self._backend.getDatasetByName("dataset1"), dataset)
+
+    def testReferenceSets(self):
+        self.assertEqual(self._backend.getNumReferenceSets(), 4)
+        referenceSets = self._backend.getReferenceSets()
+        expected_names = ["Default", "example_1", "example_2", "NCBI37"]
```

Comment:
Looks like Travis is choking on this because of the ordering. Maybe try sorting the list of reference sets by name, before running the loop below?